examples/wrapper-types/uniffi.toml

```diff
@@ -1,11 +1,58 @@
 [bindings.python]
 cdylib_name = "wrapper_types"
 
+[bindings.python.wrappers.JsonObject]
+imports = ["json"]
```

Comment:
not going to die on this hill, but I still find "imports" as too prescriptive. A common (and often significant) Python optimization here would be moving from `import json`/`json.loads(blah)` to `from json import loads`/`loads(blah)` - but this prevents me from expressing that. That optimization gets even more significant on `deep.packages.with.many.modules`

Reply:
I'd like to change `imports` to specify the entire import statement.  This lets us use the `from ... import` version on Python and I think it could be an improvement for other languages as well.